helm/utils/validate.schema.py

```diff
@@ -0,0 +1,56 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2023-2024 Open Text.
+#
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+
+import argparse
+import json
+import jsonschema
+import pathlib
+import sys
+import yaml
+
+# This should be equivalent to calling:
+#  check-jsonschema --schemafile <chart>/values.schema.json <chart>/values.yaml 
+# but gives slightly better/clearer debugging information
+
+ap = argparse.ArgumentParser()
+sub = ap.add_subparsers(title='subcommands', dest='mode', required=True)
+chart = sub.add_parser('chart', aliases=['c'], description='validate values.yaml against values.schema.json in chartpath')
+chart.add_argument('chartpath', help='path to chart to validate')
+
+files = sub.add_parser('files', aliases=['f'], description='validate values file against schema')
+files.add_argument('schema_json', help='path to json schema')
+files.add_argument('values', help='path to values.yaml/json')
+
+args = ap.parse_args()
+
+#print(args.mode)
+if args.mode.startswith('f'):
```

Comment:
Don't think this is used at present?

Reply:
Not as part of the package build automation, but it's there for use if required e.g. if you want to separately validate a user provided values.yaml against the chart instead of the default provided values.